scenarios/scenario_tests/src/test/java/intellij_generate/scenarios/S01OneClassTest.java

```diff
@@ -22,11 +22,11 @@ public static void before_all() {
   public void source_folders() {
     // this scenario has one source file, in a non-maven-standard layout
     assertEquals(asList(
-      "file://$MODULE_DIR$/../../../../01_one_class/./out/production/generated",
```

Comment:
@daicoden Note especially, scenario test changes from here on down - these call out the iml and modules.xml changes that this branch ended up prompting.

I think the fact that the old style ended up pointing to content roots that were themselves symlinks probably was confusing the idea Git integration code (or perhaps, Git itself, I don't know).